cmd/listen.go

```diff
@@ -52,19 +53,23 @@ func processListener(interfaces *[]string, lp []string, promisc bool, bpf_filter
 		*interfaces = append(*interfaces, iname)
 
 		netif, err := net.InterfaceByName(iname)
+
 		if err != nil {
 			log.Fatalf("Unable to get network index for %s: %s", iname, err)
 		}
 		log.Debugf("%s: ifIndex: %d", iname, netif.Index)
 
+		// has broadcast capabilities, so promisc = true
+		hasBroadcast := (netif.Flags&net.FlagBroadcast != 0)
```

Comment:
the purpose of parens is to make clear the eval order.  so in this case should `&` or `!=` be evaluated first?

Reply:
whops! sorry I didn't get the message. Now should be better